lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -59,11 +60,13 @@ def create_vm(attrs)
             end
             attrs[:memory_policy] = OvirtSDK4::MemoryPolicy.new(:guaranteed => attrs[:memory]) if attrs[:memory].to_i < Fog::Compute::Ovirt::DISK_SIZE_TO_GB
             attrs[:high_availability] = OvirtSDK4::HighAvailability.new(:enabled => attrs[:ha] == "1") if attrs[:ha].present?
+            
+            if attrs[:clone] == true && attrs[:disks].present?
```

Comment:
Is there an option where `:clone` is set to `false`?

Reply:
It can be nil